sdk/python/packages/flet/src/flet/controls/base_control.py

```diff
@@ -45,30 +45,40 @@ def skip_field():
 
 @dataclass_transform()
 def control(
-    cls_or_type_name: Optional[Union[type[T], str]] = None,
+    dart_widget_name: Optional[Union[type[T], str]] = None,
     *,
     isolated: Optional[bool] = None,
     post_init_args: int = 1,
     **dataclass_kwargs,
 ) -> Union[type[T], Callable[[type[T]], type[T]]]:
-    """Decorator to optionally set 'type' and 'isolated' while behaving like @dataclass.
-
-    - Supports `@control` (without parentheses)
-    - Supports `@control("custom_type")` (with optional arguments)
-    - Supports `@control("custom_type", post_init_args=1, isolated=True)` to
-        specify the number of `InitVar` arguments and isolation
+    """
+    Decorator to optionally set widget name and 'isolated' while behaving
+    like @dataclass.
+
+    Parameters:
+        dart_widget_name: The name of widget on Dart side.
+        isolated: If `True`, marks the control as isolated. An isolated control
+            is excluded from page updates when its parent control is updated.
+        post_init_args: Number of InitVar arguments to pass to __post_init__.
+        **dataclass_kwargs: Additional keyword arguments passed to @dataclass.
```

Comment:
<img width="414" height="79" alt="image" src="https://github.com/user-attachments/assets/3bc68adc-8536-41ca-ba77-f19ba3049d83" />

can you try wrapping all instances of @dataclass in backticks? mkdocs seems to parse it to a github account.
https://v1-move-update-behavior.flet-docs.pages.dev/types/control/?h=control#flet.control